esignet-core/src/main/java/io/mosip/esignet/core/validator/ClientNameLangValidator.java

```diff
@@ -0,0 +1,27 @@
+package io.mosip.esignet.core.validator;
+
+import javax.validation.ConstraintValidator;
+import javax.validation.ConstraintValidatorContext;
+import java.util.Locale;
+import java.util.Set;
+
+public class ClientNameLangValidator implements ConstraintValidator<ClientNameLang, String> {
+
+    @Override
+    public void initialize(ClientNameLang constraintAnnotation) {
+    }
+
+    @Override
+    public boolean isValid(String value, ConstraintValidatorContext context) {
+        Locale[] availableLocales = Locale.getAvailableLocales();
```

Comment:
Instead of initializing this array every time, can we initialize it once and reuse it? It can be static also.

Reply:
sure anusha